### DISCUSSION TITLE
Is there a way to disable shortcut creation?

### DISCUSSION THREAD

**Discussion by rlnt** [martinet101/ElevenClock#258] [Q&A] [answered]
Edit: right after submitting this, I found the issue for it, nevermind then https://github.com/martinet101/ElevenClock/issues/256

I am loving ElevenClock so far, really nice addition.
I guess you are able to disable the creation of a desktop shortcut when installing the app with the installer.

However, I used winget to install it and now, everytime an update of ElevenClock is installed, the desktop shortcut reappears. Is there a way to disable this?

I really don't mind frequent updates, don't get me wrong on this one! :D

**Comment by marticliment** [comment, marked as answer]
It will be fixed in #256 ;)
